covid_graphs/covid_graphs/scatter_plot.py

```diff
@@ -126,16 +122,18 @@ def transform(list_2d, graph_type=GraphType.Cumulative):
         return figure
 
 
-def main():
-    parser = argparse.ArgumentParser(description='COVID-19 visualization for Slovakia')
-    parser.add_argument('country_data',
-                        metavar='country_data',
-                        type=str,
-                        help=f"Protobuf file with country data")
-    parser.add_argument('simulated',
-                        metavar='simulated',
-                        type=str,
-                        help=f"Protobuf file with simulation results")
-    args = parser.parse_args()
-
-    SimulationReport(args.simulated, args.country_data).create_scatter_plot().show()
+@click.command(help='COVID-19 simulation visualization for Slovakia')
+@click.argument(
+    "country_data",
+    required=True,
+    type=click_pathlib.Path(exists=True)
+    # help=f"Protobuf file with country data")
+)
+@click.argument(
+    "simulation_protofile",
+    required=True,
+    type=click_pathlib.Path(exists=True)
+    # help=f"Protobuf file with simulation results")
+)
```

Comment:
You can use named arguments to have help messages :) At work we always used named args (and that's what I'd prefer), in my previous PR I did not do that though since you were worried about too much typing.

Reply:
I don't understand how to do this. I'll merge this and then you can show me :smile:

Reply:
You can easily google it, it is `@click.argument` (positional) vs. `@click.option` (named). Syntax example is in `server.py`.

Reply:
Right. But why doesn't click.argument have `help` available?